packages/modules/imodel-browser/src/containers/ITwinGrid/useITwinFavorites.ts

```diff
@@ -86,7 +88,7 @@ export const useITwinFavorites = (
           },
         });
 
-        if (!result || result.status !== 200) {
+        if (!result || (result.status !== 200 && result.status !== 204)) {
```

Comment:
this is the only api that can return a 204, https://developer.bentley.com/apis/itwins/operations/remove-itwin-from-my-favorites/

Reply:
You're right according to the docs, but when I test the add to favorites API, I get a 204 status.
I removed the 204 check from the getITwinFavorites function cuz it was indeed not needed there.
<img width="577" alt="image" src="https://github.com/user-attachments/assets/92651a85-90b1-4a7d-a8ce-16fe385df081">
<img width="842" alt="image" src="https://github.com/user-attachments/assets/fd67b3c5-0421-4c10-945e-adebdd1b7cfd">

Reply:
@davidhjones from the itwins api to comments on this

Reply:
@aruniverse can we resolve this since we're not getting a response from David?